doc/templates/uno/styles/main.css

```diff
@@ -1,10 +1,11 @@
 background-color 0.2s ease-in-out}.front-page .button.white a{background:#fff;color:#242424}.front-page .button.white a:hover{background:#7a67f8;color:#fff}.front-page .button.turquoise a{background:#7a67f8;color:#fff}.front-page .button.turquoise a:hover{background:#654ff7}.front-page .intro{padding:100px 15px 60px;text-align:center;font-size:18px;position:relative}@media screen and (min-width: 768px){.front-page .intro{padding:200px 15px 120px}}.front-page .intro h1{font-size:36px;text-transform:none;font-weight:600;width:auto;max-width:100%;margin:0 auto;letter-spacing:normal}@media screen and (min-width: 768px){.front-page .intro h1{font-size:44px}}.front-page .intro p{width:100%;font-size:22px}.front-page .intro strong{font-weight:bold}.front-page .intro .button a{margin-top:20px}.front-page .headerboxes{display:flex;flex-wrap:wrap;padding:40px 15px;background-color:#f4f2f3}@media screen and (min-width: 768px){.front-page .headerboxes{padding:80px 15px}}.front-page .headerboxes .headerbox{margin:40px auto}.front-page .headerboxes .headerbox-inner{padding:0;height:100%}.front-page .headerboxes h2{font-size:30px;font-weight:600;text-transform:none}.front-page .headerboxes h2 .anchorjs-link{display:none}.front-page .headerboxes p{font-size:16px;font-weight:600}.front-page .headerboxes .icon{margin:0}.DocSearch-Hit-content-wrapper{height:100%}
```

Comment:
Suspicious carriage return added in middle of this line.

Reply:
Hey @carldebilly 

I don't have "^M" in my code, it seems to appear only here in github, 

 ![image](https://github.com/unoplatform/uno/assets/100726410/bd3ce2b6-a879-4e0e-b6bb-01315babb1b5)

it seems to come from the transition property in the main.scss (file that I didn't edit)
![image](https://github.com/unoplatform/uno/assets/100726410/7e9cfafd-8bf0-4ec5-83a8-113537d57db1)

Do I have to manually remove the "^M" by editing the file directly from github?